address/models.py

```diff
@@ -115,6 +115,15 @@ def _to_python(value):
         # Need to save.
         address_obj.save()
 
+    # Handle cases where coordinates passed in do not match the existing object
```

Comment:
@aalmazan could move this above the save on 116 to only save once

Reply:
Actually, it shouldn't double-save because 116 is within the `except` block. If an exception happens the `if all([latitude != address_obj.latitude, longitude != address_obj.longitude]):` would never be met and wouldn't save again.

The save would only happen on cases where there's an existing Address obj from the db that needs to get updated anyway.

Reply:
oh got it -- missed that 👍